app/components/bali/tags/tag_item/component.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Bali
+  module Tags
+    module TagItem
+      class Component < ApplicationViewComponent
+        attr_reader :text, :is_delete
+
+        def initialize(
+          text: '',
+          color: nil,
+          is_light: false,
+          size: nil,
+          type: nil,
+          rounded: false,
+          is_delete: false,
+          **options
```

Comment:
Todos los tags llevan texto, por lo que debe ser un parámetro requerido

```suggestion
          text:,
          color: nil,
          light: false,
          size: nil,
          type: nil,
          rounded: false,
          delete: false,
          **options
```

Reply:
Puede ir sólo con is_delete para que se vea como un cuadro con una cruz solamente, así que text no es obligatoria:
![image](https://user-images.githubusercontent.com/19512542/178050837-82e6b9c4-1c1f-4a4b-a710-fb990abc4ebc.png)
![image](https://user-images.githubusercontent.com/19512542/178050890-7fbf2e54-0983-4a1a-b1cb-dc1620cad24b.png)